test/functional/tests/common/lvm_common_utils.py

```diff
@@ -29,7 +30,8 @@ def run_fio_on_lvm(volumes: []):
 
```

Comment:
the run_fio_on_lvm() function doesn't seem to do anything related to LVM. it seems to be more specific to some test scenarios (not integrated to the repo yet). I suggest removing it from here and moving to a test.

Reply:
This function will be used by 3 tests  (https://github.com/Open-CAS/open-cas-linux/pull/697/files , https://github.com/Open-CAS/open-cas-linux/pull/682/files , https://github.com/Open-CAS/open-cas-linux/pull/694/files ) so placing it inside each tests would be code duplication and that's why it is implemented here.

Reply:
TBH I don't mind code duplication if it makes the code easier to read. This helper function is not generic at all and it's just another function which I need to open in order to understand what exactly the test does. Please do not compress the code.

Reply:
ok, I will change it :)